### PR TITLE
bump golang 1.18 to 1.18.8 and golang 1.19 to 1.19.3

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*
Golang had patch version releases today. Update EKS Go to base off of the latest patch version releases.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/golang/go/1.18/rpmbuild/SPECS/golang.spec [hunk 1]
```diff
@@ -108,7 +108,7 @@
 # Comment out go_prerelease and go_patch as needed
 %global go_api 1.18
 #global go_prerelease rc1
-%global go_patch 7
+%global go_patch 8
```
Comment (markapruett) [inline projects/golang/go/1.18/rpmbuild/SPECS/golang.spec:111]: What are these?

Comment (markapruett) [reply projects/golang/go/1.18/rpmbuild/SPECS/golang.spec:111]: NM. Got it.

**Review by markapruett — APPROVED** [review]

File: projects/golang/go/1.18/rpmbuild/SPECS/golang.spec (hunk 1 above)
Comment (danbudris) [reply projects/golang/go/1.18/rpmbuild/SPECS/golang.spec:111]: patch version hard-coded in the spec file. we should change them to use a variable for it that we derive from the git-tag but for the moment I'm just bumping it

**Pull request merged**